<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSlider</class>
 <widget class="QWidget" name="MusicSlider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>20</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <widget class="QFrame" name="inLine">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>9</y>
        <width>400</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#inLine
{
	background:#EBEEF5;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="outLine">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>9</y>
        <width>200</width>
        <height>4</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>5</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(30,204,148);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="currSlider">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>5</y>
        <width>10</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(30,204,148);
border-radius:5px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
